.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+    -   id: black
+        args: [--safe, --quiet]
```

Comment:
Missing a newline at EOF here, according to Github's diff view. (Not that this is a big deal, but better to fix it now than later IMO.)

Reply:
Done!